### ISSUE TITLE
add a “dashboard” to garnix.io

### ISSUE THREAD

**Issue by sellout** [garnix-io/issues#48]
I wish the garnix.io page would give me a bit of an overview, rather than just listing all the most recent builds.

Basically, just a list of the latest overall results (perhaps colorized) for the default branch on each repo I have enabled garnix for. And sorted by name, to make it easy to compare.

So that, I see something like
```
con-kitty/categorifier[088b8d61]         Failure
con-kitty/concat[68a24f40]               Failure
sellout/bash-strict-mode[b35a8027]       Success
sellout/emacs-extended-faces[fd111870]   Failure
```
and then I can click through each to see the detailed results.

**Comment by jkarni** [comment]
Yeah, I was thinking of chaging the main page a bit, to be a bit like you suggest. I.e., each repo would show up, in order from last committed to (in any branch? in the main branch?). You can then click on that repo to open a `<details>` that shows the latest say 5 commits in that repo in any branch, as well as a link to the latest build in the main branch. That combines seeing `main` and seeing the latest in a way that I think makes sense.

**Comment by sellout** [comment]
The new UI is a big improvement for this. It’s still different from the original request here, but I’m happy with where it is. Having a single page that lists all the builds for a single commit is fantastic.
